public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -182,82 +175,84 @@ figure.image-display
 
-  At the moment, we're binding to !{_an} !{_array} of strings. We do that occasionally in real applications, but
-  most of the time we're binding to more specialized objects.
-
-  Let's turn our !{_array} of hero names into !{_an} !{_array} of `Hero` objects. For that we'll need a `Hero` class.
+  At the moment, you're binding to !{_an} !{_array} of strings.
+  Most of the time you'd bind to more specialized objects. To do this, turn your !{_array}
+  of hero names into !{_an} !{_array} of `Hero` objects. For that you'll need a `Hero` class.
 
   Create a new file in the `!{_appDir}` folder called  <ngio-ex path="hero.ts"></ngio-ex> with the following code:
-  
+
 +makeExcerpt('app/hero.ts')
 
 block hero-class
   :marked
-    We've defined a class with a constructor and two properties: `id` and `name`.
+    You've defined a class with a constructor and two properties: `id` and `name`.
 
-    It might not look like we have properties, but we do. We're taking
-    advantage of a TypeScript shortcut in our declaration of the constructor parameters.
+    It might not look like you have properties, but you do. You're taking
+    advantage of a TypeScript shortcut in your declaration of the constructor parameters.
 
     Consider the first parameter:
 
   +makeExcerpt('app/hero.ts ()', 'id')
 
   :marked
     That brief syntax does a lot:
-    * Declares a constructor parameter and its type
-    * Declares a public property of the same name
-    * Initializes that property with the corresponding argument when we "new" an instance of the class
+    * Declares a constructor parameter and its type.
+    * Declares a public property of the same name.
+    * Initializes that property with the corresponding argument when you use "new" to
+    instantiate a new instance of the class.
 
 .l-main-section
 :marked
   ## Using the Hero class
-  Let's make the `heroes` property in our component return !{_an} !{_array} of these `Hero` objects.
+  Make the `heroes` property in your component return !{_an} !{_array} of these `Hero` objects.
 
 +makeExcerpt('app/app.component.3.ts', 'heroes')
 
 :marked
-  We'll have to update the template.
+  Next, update the template.
   At the moment it displays the hero's `id` and `name`.
-  Let's fix that so we display only the hero's `name` property.
+  Fix that so you display only the hero's `name` property.
 
 +makeExcerpt('app/app.component.3.ts', 'template')
 
 :marked
-  Our display looks the same, but now we know much better what a hero really is.
+  Your display looks the same, but now you've clarified what a hero is.
 
 .l-main-section#ngIf
 :marked
   ## Conditional display with NgIf
 
   Sometimes an app needs to display a view or a portion of a view only under specific circumstances.
 
-  In our example, we'd like to display a message if we have a large number of heroes, say, more than 3.
+  In the example, you can display a message if there are more than three heroes.
```

Comment:
"you can" seems misleading here. The point is not what you can do, but that you're going to do it conditionally. Maybe:

Let's change the example to display...

(I know, that uses "us", but I can't think of a clearer way to introduce what's coming next. Maybe you can!)